Makefile

```diff
@@ -798,3 +798,15 @@ help:
 	@printf "\033[1mAvailable targets for minikube ${VERSION}\033[21m\n"
 	@printf "\033[1m--------------------------------------\033[21m\n"
 	@grep -h -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+
+.PHONY: update-kubernetes-version
+update-kubernetes-version:
+	(cd hack/update/kubernetes_version && \
+	 go run update_kubernetes_version.go)
+
+.PHONY: update-kubernetes-version-pr
+update-kubernetes-version-pr:
+	(cd hack/update/kubernetes_version && \
```

Comment:
consider adding a check here, that if github token is not added user to set it.
(maybe with a link to the github page that adds the token for lazy ppl like me)

Reply:
good catch, added